apps/prairielearn/assets/scripts/question.ts

```diff
@@ -113,58 +115,15 @@ function handleStatusChange(socket: Socket, msg: StatusMessage) {
 }
 
 function fetchResults(socket: Socket, submissionId: string) {
```

Comment:
Nit: the socket parameter is no longer used in this function.

Reply:
Removed in f6d3a18. However, I'm looking at this more closely and it seems like we should be closing the socket after we get results. Let me take another pass at this.

Reply:
I added back socket closing in 4dd480a.